tests/testthat/test-gamlss.R

```diff
@@ -0,0 +1,15 @@
+
+context("gamlss")
+
+
+test_that("gamlss", {
+  d <- datasets::mtcars
+  
+  twidlr_fit <- twidlr::gamlss(d, vs ~ hp + wt, family = BI())
+  origin_fit <- gamlss::gamlss(vs ~ hp + wt, family = BI(), data = d)
```

Comment:
Please use an explicit namespace for `BI()` on above two lines. E.g., `gamlss::BI()` or appropriate.